plugins/gatsby-source-cloudinary/gatsby-node.js

```diff
@@ -13,20 +13,20 @@ const type = `TobbieCloudinaryMedia`;
 // Now I only get results for the `const DEFAULT_TYPE = "upload";`
 //const type = `CloudinaryMedia`;
 
-// 🤯. 🔌 ☑️ 🎶  = ({ 😹 }) => {
+// 🤯. 🔌 ☑️ 🎶  = ({ 🥳 }) => {
 exports.pluginOptionsSchema = ({ Joi }) => {
-  // return 😹.📖({})
+  // return 🥳.📖({})
   return Joi.object({
-    //  😹.🧶().®️().💁(`Enables`),
+    //  🥳.🧶().®️().💁(`Enables`),
     cloudName: Joi.string().required(),
```

Comment:
Here is it Queen @raae

Reply:
Looks good, create a branch from and PR towards @raae/gatsby-source-cloudinary